packages/python-contourpy/build.sh

```diff
@@ -0,0 +1,40 @@
+TERMUX_PKG_HOMEPAGE=https://contourpy.readthedocs.io/
+TERMUX_PKG_DESCRIPTION="Python library for calculating contours in 2D quadrilateral grids"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.2.1"
+TERMUX_PKG_SRCURL=git+https://github.com/contourpy/contourpy
+TERMUX_PKG_DEPENDS="python, python-numpy"
+TERMUX_PKG_PYTHON_COMMON_DEPS="wheel, meson-python, build"
+_NUMPY_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python-numpy/build.sh; echo $TERMUX_PKG_VERSION)
```

Comment:
That is an absolute abomination.
Can't you just `grep` it?

Reply:
To put a finer point on it...
Doesn't sourcing another build script here overwrite the lines above?

Reply:
Why is that command an abomination? Could you suggest any alternative command?

Reply:
No, this works fine and will only produce the result of the one variable. See other build scripts that do the same.

Reply:
> Doesn't sourcing another build script here overwrite the lines above?

No, it does not overwrite the lines above because the command in `$()` is run in a subshell[^1]. It can be verified by getting the package information with `dpkg -I <path of .deb file>`

[^1]: https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Command-Substitution

> Can't you just `grep` it?

Yes, it is possible. 

```
grep "TERMUX_PKG_VERSION=" packages/python-numpy/build.sh | cut -d '"' -f 2
```

Now there will be a fight between grep vs command substitution.

Reply:
> No, this works fine and will only produce the result of the one variable. See other build scripts that do the same.

I've never seen a build script source another build script.
What package would be an example of that?

Reply:
> What package would be an example of that?

https://github.com/termux/termux-packages/blob/ad10680a12d8da76c81ba86ea7b9204448521480/packages/python-scipy/build.sh#L10

Reply:
Oh right...
It's getting sourced in a command substitution so it's a subshell and never overwrites the top of the build script.
I guess that works fine then, and I should probably get some sleep.

Reply:
Thank you for the review.